app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,16 @@
   <title>VimGolf - real Vim ninjas count every keystroke!</title>
 
   <%= stylesheet_link_tag :all %>
+  <%= stylesheet_link_tag "960" %>
```

Comment:
it used to include all css files under `pubic/stylesheets`. i don't think it works anymore, but couldn't find anywhere in any CHANGELOG that indicated it's no longer a feature. what led me to believe that initially is that there's an ampty `all.css` file so I think after it stopped working rails started trying to serve that file, realized it wasn't there and created it.

regardless, we can probably remove that line for now. my plan is now that we're on rails 4.x we can get asset pipeline up and running, which will do the requiring for us (and also speed up serving assets!)